pandas/io/formats/style.py

```diff
@@ -1648,79 +1649,124 @@ def highlight_null(
 
             .. versionadded:: 1.1.0
 
+        props : str, default None
+            CSS properties to use for highlighting. If ``props`` is given, ``color``
+            is not used.
+
+            .. versionadded:: 1.3.0
+
         Returns
         -------
         self : Styler
+
+        See Also
+        --------
+        Styler.highlight_max: Highlight the maximum with a style.
+        Styler.highlight_min: Highlight the minimum with a style.
+        Styler.highlight_quantile: Highlight values defined by a quantile with a style.
+        Styler.highlight_between: Highlight a defined range with a style.
+
+        Notes
+        -----
+        Uses ``pandas.isna()`` to detect the missing values.
```

Comment:
why is this here? we use isna for detecting missing values by default, so this is not surprising.

Reply:
removed